exp/api/v1beta1/azuremachinepool_webhook.go

```diff
@@ -319,3 +322,45 @@ func (amp *AzureMachinePool) ValidateOrchestrationMode(c client.Client) func() e
 		return nil
 	}
 }
+
+// ValidateInstanceMix validates certain instance mix limitations are not violated
+func (amp *AzureMachinePool) ValidateInstanceMix() error {
+	const standardPrefix = "Standard_"
+	supportedFamilies := []string{"A", "B", "D", "E", "F"}
+
+	// Only up to five VM sizes can be specified.
```

Comment:
Is it 4 or 5? The comment says 5 but the logic below says `a maximum of 4...`.

```suggestion
	// Only up to four VM sizes can be specified.
```

Reply:
Instance mix is constricted to 5 total per VMSS. With the way it is implemented, the .spec.vmSize is one of them so we can only have four in the additionalVSizes. Updating comment.